backends/advanced/src/advanced_omi_backend/routers/modules/__init__.py

```
@@ -33,6 +33,7 @@
 from .system_routes import router as system_router
 from .user_routes import router as user_router
 from .websocket_routes import router as websocket_router
+from .annotation_routes import router as annotation_router
```

Ruff (0.14.14)

[error] Redefinition of unused `annotation_router` from line 22: `annotation_router` redefined here

Remove definition: `annotation_router`

(F811)